study-builder/tenants/testboston/tenant.yaml

```diff
@@ -1,12 +1,28 @@
 tenant:
   default_directory: Username-Password-Authentication
 
+databases:
+  - name: study-admin-authentication
+    strategy: auth0
+    enabled_clients:
+      - Auth0 Management API
+    is_domain_connection: false
+    options:
+      disable_signup: true
```

Comment:
This means we can only manually add users to this database, giving us fine control.